packages/components/src/toolbar/test/index.tsx

```diff
@@ -25,5 +25,13 @@ describe( 'Toolbar', () => {
 				screen.getByLabelText( 'control2', { selector: 'button' } )
 			).toBeInTheDocument();
 		} );
+
+		it( 'should render a toolbar without styles if variant has been defined', () => {
```

Comment:
Since the "without styles" part is not accurate (it still has some styles), let's maybe rephrase this to something like

```suggestion
		it( 'should apply the unstyled variant correctly via the `variant` prop', () => {
```